cmd/mkcgo/generate.go

```diff
@@ -616,3 +610,20 @@ func fnNeedErrWrapper(fn *mkcgo.Func) bool {
 func fnCalledFromGo(fn *mkcgo.Func) bool {
 	return !fn.Variadic() // cgo doesn't support variadic functions
 }
+
+// goSymName returns the Go symbol name for a C symbol name.
+func goSymName(name string) string {
+	if name == "" {
+		return ""
+	}
+	if *private {
+		return strings.ToLower(name[0:1]) + name[1:]
+	}
+	if name[0] == '_' {
+		name = name[1:]
+	}
+	if name == "" {
+		return ""
```

Comment:
Is `""` actually used? It might be better to `panic` if someone tries to translate a `_` to Go, because I imagine omitting a symbol in the generated code might cause some weird syntax errors downstream.